### PR TITLE
[JOSEPH] refactor: chg requestposts, offerposts & split out findPosts

### PR DESCRIPTION
**Pull request by ethanstrominger**
Renamed Request Posts to Pin Request Posts
Renamed Offer Posts to Cav Request Posts
Duplicated Request Posts and renamed to Find Posts - same code was used before, now it will be separate code.

There could be a possible conflict if changes to Find Requests happens before this is merged.  This is prep to get request posts using firebase.

JOSEPH - THE CHANGES YOU REQUESTED WILL GO INTO THE NEXT PULL REQUEST.  THIS IMPROVES CODE, SO CAN YOU MERGE?
## Release Checklist

- [ ] This code has unit tests
- [ ] I have a plan to verify that these changes are working as expected after deploy
- [ ] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

## Security Checklist

This PR creates, modifies, or deletes:

- [ ] API routes: API routes, parameters, or user authorization
- [ ] Authentication: Authentication mechanism
- [ ] Credentials: Server side credentials, or secrets in configuration / source code
- [ ] Cryptography: Encryption, hashing, certificates, signatures, random numbers, etc.
- [ ] User data: personal information handling, logs, error messages, etc.

### CODE REVIEW

File: web-client/src/ducks/requests/actions.ts [hunk 1]
```diff
@@ -3,37 +3,49 @@ import { IRequest, Request } from 'src/models/requests';
...
+  RESET_CAV_REQUEST_POSTS,
+  RESET_FIND_REQUEST_POSTS,
+  RESET_PIN_REQUEST_POSTS,
   RESET_SET,
   SET,
   SET_TEMP_REQUEST,
 } from './types';
 
-export const getRequestPosts = (payload: IgetRequestPosts) => (
+export const getFindPosts = (payload: IgetRequestPosts) => (
```
Comment (ashwinkjoseph) [inline web-client/src/ducks/requests/actions.ts:25]: what is a find post? How is this different from CAVRequestPosts and PINRequestPosts

File: web-client/src/ducks/requests/actions.ts [hunk 2]
```diff
@@ -3,37 +3,49 @@ import { IRequest, Request } from 'src/models/requests';
...
   dispatch({
-    type: GET_REQUEST_POST,
-    firebase: getOpenPostFunc,
+    type: GET_FIND_POSTS,
+    firebase: getFindPostsFunc,
     payload,
   });
 
-export const getOfferPosts = (payload: IgetRequestPosts) => (
+export const getPinRequestPosts = (payload: IgetRequestPosts) => (
```
Comment (ashwinkjoseph) [inline web-client/src/ducks/requests/actions.ts:34]: These are the posts that the user-created as a request right? just confirming

File: web-client/src/ducks/requests/actions.ts [hunk 3]
```diff
@@ -3,37 +3,49 @@ import { IRequest, Request } from 'src/models/requests';
...
 ) =>
   dispatch({
-    type: GET_OFFER_POST,
-    firebase: getAcceptedPostFunc,
+    type: GET_PIN_REQUEST_POSTS,
+    firebase: getPinRequestPostsFunc,
+    payload,
+  });
+
+export const getCavRequestPosts = (payload: IgetRequestPosts) => (
```
Comment (ashwinkjoseph) [inline web-client/src/ducks/requests/actions.ts:43]: These are the request posts for which the user responded by making offers right? just confirming. 
Can you add this as a comment on top of the name of the function?

File: web-client/src/ducks/requests/functions.ts [hunk 4]
```diff
@@ -51,20 +51,28 @@ export const setUserRequest = async ({
...
     lng,
-    radius: 500,
-    status: AbstractRequestStatus.pending,
+    radius: 5,
+    status: AbstractRequestStatus.accepted,
   });
 
-export const getAcceptedPost = async ({ lat, lng }: IgetRequestPosts) =>
+export const getFindPosts = async ({ lat, lng }: IgetRequestPosts) =>
   functions.httpsCallable('https-api-requests-getRequests')({
```
Comment (ashwinkjoseph) [inline web-client/src/ducks/requests/functions.ts:63]: Can you switch to using firestore directly?

**Review by ashwinkjoseph — CHANGES_REQUESTED** [review]
Can you switch to using the firestore directly? this doesn't add any functional value yet, so I think we can wait for the firestore fetching to be implemented as well right?

Also what is FindPosts? if that is for the map, it doesn't need Redux, we are using Algolia SDK for it and @alexjball is taking care of it

File: web-client/src/ducks/requests/actions.ts (hunk 1 above)
Comment (ethanstrominger) [reply web-client/src/ducks/requests/actions.ts:25]: getFindPosts is used for getting posts for the map.

File: web-client/src/ducks/requests/functions.ts (hunk 4 above)
Comment (ethanstrominger) [reply web-client/src/ducks/requests/functions.ts:63]: In my next pull request.

File: web-client/src/ducks/requests/actions.ts (hunk 3 above)
Comment (ethanstrominger) [reply web-client/src/ducks/requests/actions.ts:43]: I have put it into code for my next pull request.

File: web-client/src/ducks/requests/actions.ts (hunk 1 above)
Comment (ethanstrominger) [reply web-client/src/ducks/requests/actions.ts:25]: This function was doing double duty - it was called both by find and by my requests.  It is no longer needed by my requests.  I can delete it in the next pull request.

**Comment by ashwinkjoseph** [comment]
> Can you switch to using the firestore directly? this doesn't add any functional value yet, so I think we can wait for the firestore fetching to be implemented as well right?
> 
> Also what is FindPosts? if that is for the map, it doesn't need Redux, we are using Algolia SDK for it and @alexjball is taking care of it
> 
> Could you also add a comment on top of the action creators for CAVRequestPosts and PINRequestPosts as to what they mean? I know we agreed upon a terminology, but it's not very clear to a new reader and we can't expect them to go back to reading a glossary all the time.

As per the replies in the thread on this review, these changes, as well as the changes requested on this PR, are available in the new PR. So, I am closing this for now and will be merging the new PR instead

**Pull request closed without merging**